inspirehep/modules/theme/static/scss/base/header.scss

```diff
@@ -748,22 +748,30 @@ a.external-link::after {
 
                 #tools-dropdown i,
                 #dropdown-links-stats i,
-                li:nth-child(3) a:before,
-                li:nth-child(4) a:before {
+                li:nth-child(1) a:before,//changed number of child from 3
```

Comment:
Why was the change needed? Is it working both with INSPIRE_FULL_THEME True and False?

The comment does not give information.